### PR TITLE
Tribute page

### PR DESCRIPTION
**Pull request by camilevahviraki**
### **In This tribute page**

- **We added Index.html file**   
  
- **added some style.css**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        </div>
+        <div class="img-div">
+            <img id="img-caption" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="photography">
+            <h3>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part
+                of his life-long war on hunger.</h3>
+        </div>
+        <div class="p">
+            <h2>Here's a time line of Dr. Borlaug's life:</h2>
+        </div>
+        <div class="list">
```
Comment (camilevahviraki) [inline index.html:25]: Please use semantic tags Instead of <div> tag . You should use <main> tag instead of <div id=main>

**Review by camilevahviraki — COMMENTED** [review]
> **Required** **changes**:

- [ ] Please use semantics tags, they make your code better.
- [ ] Edit your README.md file

> **Optional changes**:

- [ ] Use [linters](URL "https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md") to remove errors that you may you could not have detected

  Thanks

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:25]: Good job 👍 in identifying the proper use of semantic tags.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @camilevahviraki, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job identifying readme should be professional.
- Good job 👍 in providing link to linters.

## Required Changes ♻️
- Linters are not optional but it's required. Kindly make sure to mention that Linters are required to setup the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (camilevahviraki) [inline index.html:25]: Please use semantics tags instead of `<div >`, You may use `<main>` at the place of`<div id="main">`

**Review by camilevahviraki — COMMENTED** [review]
Hi Micronaut, :wave:

Good job so far!:+1: 
There are some issues that you still need to work on but you are almost there!
> ### **Required Changes** :recycle: 

- [ ] Please use semantic tags instead of `<div>` , They make your code better
- [ ] Setup [Linters](url 'https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md') in your project directory, It's can help you to remove errors that you could have not detected
- [ ] Please Edit your README.md file to make it Professional.

Thanks and Happy coding! :clap::clap::clap:

**Review by sja-thedude — APPROVED** [review]
Hi @camilevahviraki, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**